lib/resources/r4/care_team.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: CareTeam
+field_name_base_url: http://hl7.org/fhir/R4/careteam-definitions.html#CareTeam
+fields:
+- name: patient
```

Comment:
patient doesn't appear to be needed in your .md

Reply:
Removed. https://github.com/cerner/fhir.cerner.com/pull/470/commits/1c1aad456a43d4ba5ae6e7fc8620a05d19202987